### DISCUSSION TITLE
Support HLS and MPEG Dash for Radio Streaming

### DISCUSSION THREAD

**Discussion by kvnkim** [navidrome/navidrome#2903] [Ideas]
I currently use the feature to stream from what I assume are shoutcast radio stations which have simple urls, but newer internet radio streams are moving to HLS or MPEG Dash. Would this be possible for the radios feature?

I believe mpd and hls streams are possible to be decoded using ffmpeg?

`ffmpeg -user_agent "navidrome" -referer "https://emp.bbc.co.uk/" -i "https://a.files.bbci.co.uk/ms6/live/3441A116-B12E-4D2F-ACA8-C1984642FA4B/audio/simulcast/dash/nonuk/`

Example list of HLS/MPEG Dash streams of BBC Radio
https://en.everybodywiki.com/List_of_BBC_radio_stream_URLs#Digital-only_stations

**Comment by klausmcm** [comment]
I would appreciate this functionality as well.
